lib/tests/streamlit/runtime/state/session_state_test.py

```diff
@@ -317,6 +318,38 @@ def test_updates(self):
         assert sr5.get("slider")[0].value == 100.0
         assert sr5.get("slider")[1].value == 212.0
 
+    def test_serializable_check(self):
+        with patch_config_options({"runner.onlySerializableSessionState": True}):
+            script = self.script_from_string(
+                "unserializable.py",
+                """
+                import streamlit as st
+
+                def nested():
```

Comment:
Nit: this name kinda obscures its purpose - can we call the function "get_unserializable_data()" or something?